joplin/api/schema.py

```diff
@@ -231,6 +231,22 @@ class Meta:
         interfaces = [graphene.Node]
 
 
+class EventFilter(FilterSet):
+    order_by = OrderingFilter(
+        fields=(
+            ('date'),
+        )
+    )
+
+    class Meta:
+        model = EventPage
+        fields = {
+            'date': ['exact', 'lte', 'gte'],
```

Comment:
I think we talked about this, but adding a note/link for more info on where 'lte' and 'gte' come from, etc. so we know where to dig for more info

Reply:
I put it in the PR, do you think it should be also as a comment in the code?

Reply:
I personally wouldn't be opposed to a link to https://docs.djangoproject.com/en/3.0/ref/models/querysets/#field-lookups
, but a least a comment like 
'these 'fields' correspond to django queryset field-lookups' so future us knows where to look (and also where to _not_ look ) in docs